src/main/resources/application.conf

```diff
@@ -142,11 +142,20 @@ mantis {
       # Maximum MPT components in a single response message (as a blockchain host)
       max-mpt-components-per-message = 200
 
-      # Maximum number of peers this node can connect to
-      max-outgoing-peers = 50
+      # Minimum number of peers this node tries to connect to at all times
+      min-outgoing-peers = 20
+
+      # Maximum number of peers this node can connect to at any time
+      max-outgoing-peers = 30
```

Comment:
Maybe we can increase this nubmer to 50? At the node startup it is really important to traverse the list of discovered peers as fast as possible to try to grab as many connections as possible.

Reply:
Yes I was wondering whether the `max-outgoing-peers` should have a relation with the `max-incoming-peers`, but I guess in our case we know that we'll not achieve 50 handshaked outgoing ones. And even if we did, and `max-incoming-peers` is 30 and may get pruned back, if the `min-outgoing-peers` vs `max-incoming-peers` has an optimal (like 40/60) ratio it should over time still allow a stable network where everyone's demand is just satisfied.

Reply:
Raised to 50.